src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.vb

```diff
@@ -245,8 +245,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
 
             Select Case symbol.TypeKind
-                Case TypeKind.Class,
-                     TypeKind.Submission
+                Case TypeKind.Class
+                    partKind = If(symbol.IsStatic, SymbolDisplayPartKind.StaticClassName, SymbolDisplayPartKind.ClassName)
+                Case TypeKind.Submission
```

Comment:
>Case TypeKind.Submission [](start = 16, length = 24)

Why submission is not using new kind? For example, it looks like C# implementation treats TypeKind.Submission the same way as TypeKind.Class.

Reply:
Thanks. I was treating them the same in the CSharp visitor; I will update.